src/components/AppContent.js

```diff
@@ -1,13 +1,98 @@
 import '../css/App.css';
+import React, { useState } from 'react';
 import KevContainer from './KevContainer.js';
 import QuoteContainer from './QuoteContainer.js';
+import StartButton from './StartButton.js';
+import ItsKevButton from './ItsKevButton.js';
+import NotKevButton from './NotKevButton.js';
+import Points from './Points.js';
+import ResetButton from './ResetButton.js';
+import newQuote from '../utils/quoteGenerator';
+
+const placeholderQuote = {author: "Placeholder", quotation: "Hit the button to start."};
+const endQuote = {author: "Placeholder", quotation: "That's all folks!"};
+let usedQuotes = [];
 
 const AppContent = () => {
+  const [quote, setQuote] = useState(placeholderQuote);
+  const [status, setStatus] = useState("start");
+  const [points, setPoints] = useState(0);
+
+  if (status === "reset") {
+    setQuote(placeholderQuote);
+    setPoints(0);
+    setStatus("open");
+  }
+
+  if (status === "end") {
+    document.getElementById("its-kev-button").setAttribute("style", "display: none;");
```

Comment:
Any thoughts here on instead doing a status check _inside_ of the two button components that either shows them or not? 

Ie: 
```
return (
  { status !== "end" && (
    <Button code...>
  )}
)
```

Doing so would save the anti-pattern of doing actual JS manipulation of the DOM instead of doing it through react 😁

Reply:
I've sorted this out now so that the button components return `null` when the game status is `end`!